app/views/doc/_translations.html.erb

```diff
@@ -20,40 +20,29 @@ This book is available in
   Partial translations available in
   <table>
     <tr><td><a href="/book/cs">Čeština</a>,</td></tr>
```

Comment:
I think this file is trash that should be deleted, and the `app/views/books/_translations.html.erb` is the one that should be fixed, no?

Reply:
That would be great. We would have only one file to update.